mono/mini/interp/transform.c

```diff
@@ -7,6 +7,9 @@
  * Copyright (c) 2004.
  */
 
+#include "config.h"
+#undef MONO_INSIDE_RUNTIME /* HACK: remove me */
```

Comment:
Could you add a comment of why we need this hack?

Reply:
`config.h` was needed here, because otherwise `DISABLE_INTERPRETER` wouldn't be visible. However, with `config.h` also `MONO_INSIDE_RUNTIME` will be visible, and that causes compilation errors because the interpreter is still using some unchecked functions (e.g. `mono_ldstr`). It's something we should fix, but I didn't want to introduce additional noise in this PR.

Anyway, the `#include "config.h"` went away for some other reason, so the hack is gone. But we should still use checked versions at some point 🙂